t/op/groups.t

```diff
@@ -136,7 +136,7 @@ sub Test {
         endgrent;
         skip "No group found we could add as a supplementary group", 1
             if (!@sup_group);
-        $) = "$) @sup_group[2]";
+        $) = "$) $sup_group[2]";
```

Comment:
this seems like a bug fix - nice catch

Reply:
Yes, that's an example of the 3rd type of warnings correction from my talk yesterday, i.e., fix the problem which the warning points to.  I reincorporated this example into my talk, just in case I ever give it again.